src/ensembl/src/content/app/entity-viewer/types/slice.ts

```diff
@@ -9,6 +9,7 @@ export type Slice = {
     name: string;
     strand: {
       code: Strand;
+      value?: number;
```

Comment:
why?

Reply:
`value` is in the GraphQL schema so I added that here

Reply:
If we aren't using it, I don't think we need it in our types.

Reply:
Shouldn't our types align with the GraphQL schema?

Reply:
They would align with the graphql schema if they were automatically generated from it, but since we are writing them manually, and since we are the only consumer of our own types, there is no point in our describing fields that we do not use and thus do not really care about.

Reply:
✔️